core/node/proof_data_handler/Cargo.toml

```diff
@@ -10,13 +10,26 @@ keywords.workspace = true
 categories.workspace = true
 
 [dependencies]
+vm_utils.workspace = true
 zksync_config.workspace = true
 zksync_dal.workspace = true
 zksync_object_store.workspace = true
 zksync_prover_interface.workspace = true
+zksync_tee_verifier.workspace = true
 zksync_types.workspace = true
-
-tracing.workspace = true
+zksync_utils.workspace = true
 anyhow.workspace = true
 axum.workspace = true
 tokio.workspace = true
+tracing.workspace = true
+
+[dev-dependencies]
+http-body-util = "0.1.0"
```

Comment:
Also this dependency seems to be unused.

Reply:
Brain it is 🤖 
Whenever I see a new dep, I check a) how (if) it's used, and b) whether we can avoid that.

Reply:
OTOH [cargo-udeps](https://github.com/est31/cargo-udeps) exists, but last time I used it it had plenty of false negatives. It doesn't handle workspace deps well, I guess.

Reply:
[Fixed](https://github.com/matter-labs/zksync-era/pull/1993/commits/a96bf714791796307aab5fe2733267af78a138d5) ✅